tests/e2e/config/jest.config.js

```diff
@@ -5,13 +5,33 @@ const { jestConfig } = require( '@automattic/puppeteer-utils' );
 config( { path: path.resolve( __dirname, '.env' ) } );
 config( { path: path.resolve( __dirname, 'local.env' ) } );
 
+// Define paths to look for E2E tests.
+const e2ePaths = {
+	wcpay: path.resolve( __dirname, '../specs/wcpay' ),
+	subscriptions: path.resolve( __dirname, '../specs/subscriptions' ),
+	blocks: path.resolve( __dirname, '../specs/blocks' ),
+};
+
+// Allow E2E tests to run specific tests - wcpay, subscriptions, blocks, all (default).
+const allowedPaths = [];
+if ( process.env.E2E_GROUP ) {
+	if ( process.env.E2E_BRANCH ) {
+		allowedPaths.push(
```

Comment:
When trying to supply unknown values for `E2E_GROUP` and `E2E_BRANCH`, we get the following error:
```
● Validation Error:

  Directory /Users/deepakpathania/Work/woocommerce-payments/undefined/shopper1 in the roots[0] option was not found.

  Configuration Documentation:
  https://jestjs.io/docs/configuration.html
```
Do you think we should handle it explicitly and mention that the one of the allowed values should be specified with the list of those?

Reply:
I'll look into this :)

Reply:
Done. Added exception if group or branch is invalid